### PR TITLE
[MODULAR] [FIX] Fixes lewd chems not changing gender when only genderswapping is enabled

### PR DESCRIPTION
**Pull request by Gofawful5**
## About The Pull Request

I didn't think about this very much until it was brought up, but if genderswapping is enabled, succubus milk and incubus draft will change gender, regardless of if enlargement chems are enabled or not.

## How This Contributes To The Skyrat Roleplay Experience

Fixes something that should've worked earlier.

## Changelog

:cl: Goffy
fix: Fixes succubus milk / incubus draft not changing gender when only genderswapping is enabled.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents.dm [hunk 1]
```diff
@@ -229,6 +229,14 @@ Primarily used in reagents/reaction_agents
 
 /// Called when an overdose starts
 /datum/reagent/proc/overdose_start(mob/living/M)
+	///SKYRAT EDIT ADDITION: Because these chemicals shouldn't bear the same weight as normal / debatably more harmful chemicals.
+	if(name == "dopamine")///This one also shouldn't have any negative mood effect.
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents.dm:233]: this is not how you do this in byond

File: modular_skyrat/modules/customization/modules/surgery/organs/genitals.dm [hunk 2]
```diff
@@ -94,7 +94,7 @@
 			if(SHEATH_NORMAL)
 				returned_string = "You see a sheath."
 			if(SHEATH_SLIT)
-				returned_string = "You see genital slit."
+				returned_string = "You see a slit." ///Typo fix.
```
Comment (Iamgoofball) [inline modular_skyrat/modules/customization/modules/surgery/organs/genitals.dm:97]: atomize this

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/chemistry_for_ERP.dm [hunk 3]
```diff
@@ -74,6 +74,13 @@
 * APHRODISIACS
 */
 
+///To notify, but not harm the player in terms of mood. Used for camphor, pentacamphor, succubus milk, and incubus draft.
```
Comment (Iamgoofball) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/chemistry_for_ERP.dm:77]: it's an overdose, it should harm the mood

**Comment by Iamgoofball** [comment]
I need to check the PR logs, but I'm positive we intentionally removed the chemicals adjusting the mob's gender.

File: code/modules/reagents/chemistry/reagents.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/reagents/chemistry/reagents.dm:233]: also, no, this is an undocumented change in a PR that makes zero mention of it, absolutely not, atomize this

**Comment by nikothedude** [comment]
holy shit tracy i remember when you just played a funny catgirl on manuel this is too far

i fear for your soul

File: code/modules/reagents/chemistry/reagents.dm (hunk 1 above)
Comment (Gofawful5) [reply code/modules/reagents/chemistry/reagents.dm:233]: > this is not how you do this in byond

it literally worked idk what to tell u

**Comment by Gofawful5** [comment]
> I need to check the PR logs, but I'm positive we intentionally removed the chemicals adjusting the mob's gender.

wat. i literally made this pr earlier and im fixing it a lil
i think thats dumb n stupid because it Specifically checks for prefs

**Comment by Gofawful5** [comment]
> holy shit tracy i remember when you just played a funny catgirl on manuel this is too far
> 
> i fear for your soul

i already made this pr im just fixing thign

**Comment by Gofawful5** [comment]
wait a goddamn second

**Pull request closed without merging**